inc/commondbtm.class.php

```diff
@@ -2044,14 +2044,17 @@ function showFormButtons($options=array()) {
       $params['candel']       = true;
       $params['canedit']      = true;
       $params['addbuttons']   = array();
+      $params['formfooter']   = null;
```

Comment:
Shouldn't this be "true" by default to keep current behavior, and allow to use "false" to disable it where needed ?